assets/javascripts/modules/analytics/omniture.js

```diff
@@ -51,6 +53,18 @@ define([
         s.eVar19 = 'D=c19';
         s.prop42 = 'Subscriber';
         s.prop47 = 'UK';
+
+        var main = $('main')[0];
```

Comment:
1) Agreed, actually this is additional data for the page load so I should really add these to the body tag. 

2) Just used map for better browser support, but thats just an old mindset of mine as forEach it pretty well supported. 

3) Agree, I'm just adding to the magic here, I suggest another card to do a mapping from a structured descriptive JSON object to the magic propXX numbers.